event_routing_backends/processors/xapi/event_transformers/problem_interaction_events.py

```diff
@@ -133,6 +132,17 @@ class ProblemSubmittedTransformer(BaseProblemsTransformer):
     """
     additional_fields = ('result', )
 
+    def get_object(self):
```

Comment:
I needed to make this change because these events were failing at the LRS level for not being IRIs, you can see the change in the test for what it's doing.